crates/hyperqueue/src/client/commands/journal/mod.rs

```diff
@@ -21,28 +21,37 @@ pub struct JournalOpts {
 
 #[derive(Parser)]
 enum JournalCommand {
-    /// Export events from a journal to NDJSON (line-delimited JSON).
-    /// Events will be exported to `stdout`, you can redirect it e.g. to a file.
+    /// Export events from a journal file
+    ///
+    /// Events are exported as NDJSON on `stdout`.
     Export(ExportOpts),
 
-    /// Stream events from a running server, it first replays old events
-    /// then it waits for new live events.
+    /// Replays all old events, then streams new events
```

Comment:
We should synchronize "all old" and "old" below, it looks like there's a difference.